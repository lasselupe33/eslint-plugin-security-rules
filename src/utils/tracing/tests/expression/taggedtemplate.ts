function myTag(strings: TemplateStringsArray, age: number) {
  const str0 = strings[0]; // "That person is a"
  const str1 = strings[2]; // "."

  let ageStr;
  if (age > 99) {
    ageStr = "old";
  } else {
    ageStr = "young";
  }

  return `${str0}${ageStr}${str1}`;
}

const age = 43;
const output = myTag`That person is a${age}.`;

const start = output;

export {};
