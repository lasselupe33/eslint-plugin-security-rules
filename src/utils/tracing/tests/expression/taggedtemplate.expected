That person is a
old
youngThat person is a.